### ISSUE TITLE
Incompatability with Redux 4 in react-redux-subspace/redux-subspace

### ISSUE THREAD

**Issue by carl-berg** [ioof-holdings/redux-subspace#95]
I am trying to used `subspaced` from the `react-redux-subspace`  and am currently getting an error

```
...my-project-path/node_modules/redux-subspace/src/index.d.ts
(34,21): Namespace '"...my-project-path/node_modules/redux/index"' has no exported member 'GenericStoreEnhancer'.
```

```ts
<Route
  path="something"
  component={subspaced((state) => state["module-namespace"], "module-namespace")(Component)}/>
```
### My Setup
| package        | version(s)
| ---------------| -------
| redux          |  4.0.0 |
| redux-subspace | 2.5.0 |
| react-redux-subspace | 2.5.0 |

Not sure if this is already known and part of #87 or not, but if you are aware you can feel free to just close this issue

**Comment by mpeyper** [comment]
Yes, I was aware the types are not compatible, but I do not believe there was ever an issue to capture it.

They have been fixed in master, as part of the redux-observable v1 compatability changes , but have not been released yet as not all packages have been confirmed to be compatible with redux 4 yet.

I'll keep this open until a new version goes out, just in case anyone else comes looking.

**Comment by mpeyper** [comment]
Fixed in #97
